src/docs/product/issues/issue-details/performance-issues/slow-db-queries/index.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Slow DB Queries'
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/slow-db-queries/
+description: 'Learn more about Slow DB Query issues and how to diagnose and fix them.'
+---
+
+<Include name="early-adopter-note.mdx" />
+
+Slow DB Query issues are created when a particular `SELECT` SQL query in your application consistently takes longer than `1000ms` to resolve.
+
+## Detection Criteria
+
+The detector for this performance issue looks at singular database spans with the following criteria:
+
+- The span operation starts with `db` and the query is stored within the span description
+- The query is specifically a `SELECT` query
+- The span duration is at least `1000ms`
+
+Keep in mind that an issue will not be created unless the query in question is _consistently_ taking longer than `1000ms`. More specifically, the issue is created only if the same query is detected as a slow query
+in 100 events within a 24 hour period.
+
+## Span Evidence
+
+The "Span Evidence" section of a Slow DB Query issue contains the following fields:
+
+- Transaction name
+- Slow DB Query
+- Duration impact - The duration of the query with respect to the entire transaction, to put into perspective how much of the transaction is being taken up by the query
+
+![Slow DB Query span evidence](span-evidence.png)
+
+View it by going to the **Issues** page in Sentry, selecting your project, clicking on the Slow DB Query issue you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.
+
+## Fixing Slow DB Queries
+
+One common cause of slow queries is that the table that is being queried from is un-indexed, so it is recommended to confirm that your table is properly indexed first and foremost. Aside from that, there's no immediately obvious way to fix slow queries, but there are some steps you can take to narrow down what specifically is causing your query to be slow.
+
+If it is supported by your version of SQL, you can run an `EXPLAIN` command on your query which will present you with a query plan. The plan will lay out the individual steps that your database will perform in order to execute the query, along with accompanying information with each step, such as its cost. The cost is an arbitrary value, but it will be a higher value depending on how "expensive" that portion of the query is. You can contrast the cost of each step to determine which steps of your query are the most expensive. Additionally, you can also run `EXPLAIN ANALYZE`, which will execute the query and report back with the same information as a regular `EXPLAIN`, but also with the actual time taken by each step of your query (be careful not to run this on any destructive queries, as they _will_ be fully executed!).
```

Comment:
### 
```suggestion
If it's supported by your version of SQL, you can run an EXPLAIN command on your query. This will present you with a query plan. The plan will lay out individual steps that your database will perform in order to execute the query, along with additional information, such as its cost. 

The cost is an arbitrary value, but it will fluctuate based on how “expensive” that portion of the query is. You can compare the cost of each step to determine which ones are the most expensive. Additionally, you can run EXPLAIN ANALYZE, which will execute the query and report back with the same information as a regular EXPLAIN, plus the actual time taken by each step of your query (be careful not to run this on any destructive queries, as they /will/ be fully executed!).
```

Reply:
I kept your suggestion but restored the markdown symbols for code and italics if that's okay! It helps out with the readability of the documentation 😄 

```suggestion
If it's supported by your version of SQL, you can run an `EXPLAIN` command on your query. This will present you with a query plan. The plan will lay out individual steps that your database will perform in order to execute the query, along with additional information, such as its cost. 

The cost is an arbitrary value, but it will fluctuate based on how “expensive” that portion of the query is. You can compare the cost of each step to determine which ones are the most expensive. Additionally, you can run `EXPLAIN ANALYZE`, which will execute the query and report back with the same information as a regular `EXPLAIN`, plus the actual time taken by each step of your query (be careful not to run this on any destructive queries, as they _will_ be fully executed!).
```

Reply:
Absolutely!